provers/groth16/arkworks-adapter/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+#[cfg(test)]
+mod integration_tests;
+
+use ark_ff::PrimeField;
+use ark_relations::r1cs::{ConstraintSystemRef, Field};
+use lambdaworks_groth16::{common::*, r1cs::R1CS, QuadraticArithmeticProgram};
+use lambdaworks_math::traits::ByteConversion;
+
+use std::ops::Deref;
+
+pub fn to_lambda<F: PrimeField>(
+    cs: &ConstraintSystemRef<F>,
+) -> (QuadraticArithmeticProgram, Vec<FrElement>) {
+    (
+        QuadraticArithmeticProgram::from_r1cs(r1cs_from_arkworks_cs(cs)),
+        extract_witness_from_arkworks_cs(cs),
+    )
+}
```

Comment:
ark_cs_to_lambda_cs would be a better that name

Also I'd document with /// what this is doing, the QAP is clear, but you need to check the code to realize the second parameter is the witness